contents/handbook/engineering/development-process.md

```diff
@@ -1,54 +1,27 @@
-  * who / the scale of the customer you're building for
-  * if you can get from your hacky MVP to production ready easily (it's ok to start with basic, but be mindful of making it harder to fully roll something out in future)
-  * if you know what you're doing or need someone from another team's expertise to get the right architecture or overall approach. We've lots of experienced people, get their help if you would benefit from it.
-* **Break into goals:** The owner turns this ambiguous milestone into a roadmap of ambitious meaningful sprint sized goals, thinking 2 - 3 sprints ahead to give other functions time
-* **Execute on goals:** Our sprint planning session is where the high level goals for the next sprint get decided. Each team takes these and breaks them down and adds their own smaller items like bugfixes.
-  The sprint planning works like this:
-     * During the week of the sprint planning, teams (with the help of the product managers) define one or two big ambitious goals for this sprint that align with major milestones.
-     * On Wednesday we get together. We use Zoom's breakout rooms to split into the small teams, and each team finalizes the ambitious goals that they want to tackle.
-     * We come back in the same room, where
-        * We do a quick retrospective on the goals for the last sprint (did we get everything done? Should we have prioritized other things?)
-        * Each team lead presents the main goals for their team. We have an open discussion about whether the goals are the right goals for this sprint and whether they are ambitious enough.
-        * Each goal should have a single owner.
-    * This meeting is also a chance to coordinate any cross-team dependencies. These should be the exception, not the rule. 
-    * As one of our values is [stepping on toes](/handbook/company/values#step-on-toes), during the sprint you might come across something that should be much higher priority than what was already planned. It's up to you to then decide to work on that as opposed to what was agreed in the sprint planning session.
-* **Evaluate success:** Review impact of each major milestone and feedback into the planning process
+To start, Product and Engineering should align on major milestones (e.g. Collaboration) which we have strong conviction will drive our success metrics for a feature.
 
-**What about the small stuff?**
+There are two types of goals.
 
-Everything can’t directly contribute to a company level goal, it’s important that the small stuff also gets done for us to succeed.
+* **Moonshots:** These are big, scary goals where we expect to fail 50% of the time. If we fail we expect to learn something equally as valuable as if we succeed. Just scraping the goal counts as a success.
```

Comment:
I know this wasn't your writing so feel free to ignore for this PR, but do we really go for moonshots, given the individual products we're building already have product-market fit? Even data warehouse, which is a very ambitious product, I'd consider a roofshot given everyone knows what data warehouses are, they already exist etc.

Reply:
I defer to the VP of Engineering, aka @benjackwhite

Reply:
Narrator: But PostHog doesn't actually do moonshots.

Reply:
Let's merge this in now with Joe's edits and then spin up a separate PR on the whole moonshots thing, but yeah I agree it doesn't jive with our actual product strategy.